app/controllers/people_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    @people_hash = {}
+    @people.each do |person|
+      @tasks_completed = person.tasks.where(completed: true)
+      @people_hash[person.name] = @tasks_completed.count
+    end
+    @people_hash
```

Comment:
Since you are only using `@people_hash` in your view, you should not being using instance variables for `@people` and `@tasks_completed` since they are just used here in your controller logic.